package.json

```diff
@@ -11,7 +11,8 @@
     "pretest": "npm run lint",
     "test": "npm run jest",
     "jest": "jest",
-    "start": "node src"
+    "start": "node src",
+    "debug": "nodemon server.js"
```

Comment:
I think `debug` is too vague, since we are potentially going to have multiple entry points/apps running at the same time.  Should this be `web` and `web-debug` or `server` and `server-debug`?

Also, you've removed `src/index.js` (by accident I hope?), and we should think about integrating the code in there with the web server startup.  See my comment below on this.

Reply:
This is still outstanding...